src/selector.rs

```diff
@@ -415,9 +417,9 @@ impl Period {
     }
 }
 
-fn has_selected_tags(actual: &HashSet<String>, requested: &HashSet<String>) -> bool {
+fn has_selected_tags(actual: &HashSet<String>, requested: &HashSet<Atom>) -> bool {
```

Comment:
Hey @Yoric what is your plan on using ```Atoms``` - do you think we should use atoms for taxonomy-owned data only (e.g. service and service's getters and setters tags are atoms, but service selectors aren't), or API consumers should provide atoms as well (service selector tags also should be atoms)?